images/ipfailover/keepalived/lib/config-generators.sh

```diff
@@ -59,6 +59,11 @@ function generate_script_config() {
   if [ "$port" = "0" ]; then
     echo "   script \"true\""
   else
+    if [[ -n $HA_CHECK_SCRIPT ]]; then
+      echo "   if [[ -f $HA_CHECK_SCRIPT ]]; then"
```

Comment:
Are you generating a script here? This is pretty insane. This expansion of `$HA_CHECK_SCRIPT` will happen at `echo` time, not at runtime.